code/frontend/MakeItSo/MakeItSo/MakeItSo.docc/Tutorials/Building a simple todo list UI.tutorial

```diff
@@ -0,0 +1,473 @@
+      * Add a _Cancel_ button
+      * Add a title to the dialog
+      
+      @Image(source: placeholder-image.png, alt: "")
+    }
+    
+    @Steps {
+      You might have noticed that the input field was not active when the dialog appears. This is inconvenient, especially on larger phones, when the input field might be hard to reach. In the next few steps, you will use SwiftUI's `@FocusState` property wrapper to place the cursor in the first input field when the dialog appears.
+      @Step {
+        Go back to the `AddReminderView.swift` file, and define an enum `FocusableField` whith a single case `title`.  
+        
+        @Code(name: "AddReminderView.swift", file: building-todo-ui-reminder-06-addreminder-brushup-01.swift, previousFile: building-todo-ui-reminder-04-addreminderview-10.swift)
+      }
+      
+      @Step {
+        Add a new private property `focusedField` as an optional `FocusedField` and mark it using the `@FocusState` property wrapper.  
+        
+        @Code(name: "AddReminderView.swift", file: building-todo-ui-reminder-06-addreminder-brushup-02.swift)
+      }
+      
+      @Step {
+        Use the `focused` view modifier to bind the focus state `focusedField` to the `TextField`. By using the `.title` enum case in the equality check, you tell SwiftUI to focus this field whenever `focusedField` is equal to `.title`.  
+        
+        @Code(name: "AddReminderView.swift", file: building-todo-ui-reminder-06-addreminder-brushup-03.swift)
+      }
+      
+      @Step {
+        In the final step, add an `onAppear` view modifier to the `Form` and set the `focusedField` property to `.title`.
+        
+        > Note: This makes sure the cursor will be placed inside the _Title_ `TextField` whenever the view appears.
+        
+        @Code(name: "AddReminderView.swift", file: building-todo-ui-reminder-06-addreminder-brushup-04.swift)
+      }
+      
+      In the next step, you will make sure the _Add_ button is only active if the title of the reminder is not empty. This prevents adding empty reminders.
+      
+      @Step {
+        Add a `disabled` view modifier to the _Add_ button and bind it to the the `isEmpty` state of the `title` property of the reminder. 
+        
+        @Code(name: "AddReminderView.swift", file: building-todo-ui-reminder-06-addreminder-brushup-05.swift) {
+          @Image(source: building-todo-ui-reminder-06-addreminder-brushup-05.png, alt: "")
+        }
+      }
+      
+      @Step {
+        Add the `navigationTitle` view modifier to the `Form` to set the dialog title, and set the  `navigationBarTitleDisplayMode` to `.inline`.
+        
+        @Code(name: "AddReminderView.swift", file: building-todo-ui-reminder-06-addreminder-brushup-06.swift) {
+          @Image(source: building-todo-ui-reminder-06-addreminder-brushup-06.png, alt: "")
+        }
+      }
+      
+      @Step {
+        Add a private function `cancel`, and call `dismiss()` to dismiss the dialog. 
```

Comment:
This step and the step below (steps 7 and 8) are not being highlighted in the code. Possibly because the file used in `@Code` is the same?

Reply:
Yes, that's the reason. Thanks for catching this!